app/src/pages/inside/dashboardPage/widgets/components/testsTableWidget/pTypes.js

```diff
@@ -1,4 +1,5 @@
 import { string, node, shape, arrayOf, bool, number, oneOf } from 'prop-types';
+import { FAILED, PASSED, SKIPPED } from '../../../../../../common/constants/launchStatuses';
```

Comment:
We have alias for 'common' folder. No need of ../../../../../../
Please fix all following cases.

Reply:
Fixed